test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -538,23 +570,26 @@ def func(x):
             return x + torch.full(x.shape, torch.tensor(torch.finfo(x.dtype).min))
 
         self.run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
-            func, (torch.randn(3, 4),)
+            func,
+            (torch.randn(3, 4),),
         )
 
     def test_operator_with_scalar_output(self):
         def func(x, y):
             return x.item() + y
 
         self.run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
-            func, (torch.tensor([1]), torch.randn(3, 4))
+            func,
+            (torch.tensor([1]), torch.randn(3, 4)),
         )
 
     def test_operator_with_dynamic_output_shape(self):
         def func(x):
             return x.nonzero()
 
         self.run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
-            func, (torch.randn(3, 4),)
+            func,
+            (torch.randn(3, 4),),
```

Comment:
nit: majority of diff is the style change, would appreciate reverting but this is minor.

Reply:
I've actually tried, but when I ran `lintrunner -a`, it changed back again ;/